packages/web5-agent/src/web5-agent.ts

```diff
@@ -54,6 +58,28 @@ export type DwnResponse = {
   reply: UnionMessageReply;
 };
 
+/**
+ * TODO: add JSDoc
+ */
+export type ProcessVcRequest = {
```

Comment:
May need to add `dataStream?: ...` here once you move the data to blob conversion into `VcApi`